### PR TITLE
Elytra recast module

### PR DESCRIPTION
**Pull request by Pivo1lovv**
Recasts elytra while holding jump key
Works on some grim servers

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleElytraRecast.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    init {
+        enableLock()
+    }
+
+    fun castElytra(player: ClientPlayerEntity): Boolean {
+        return if (checkElytra(player) && checkFallFlyingIgnoreGround(player)) {
+            player.networkHandler?.sendPacket(
+                ClientCommandC2SPacket(player, ClientCommandC2SPacket.Mode.START_FALL_FLYING)
+            )
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleElytraRecast.kt]: network.sendPacket(...)

File: src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinLivingEntity.java [hunk 2]
```diff
@@ -182,6 +190,38 @@ private void hookAirJump(CallbackInfo callbackInfo) {
         }
     }
 
+    private ClientPlayerEntity player() {
+        return MinecraftClient.getInstance().player;
+    }
```
Comment (1zun4) [inline src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinLivingEntity.java]: You are already working with the client player entity instance inside the [MixinLivingEntity]. You can therefore use `this` instead.

**Comment by Pivo1lovv** [comment]
Should be good now. Sorry for the bad implementation, this is my first time working with fabric and kotlin

**Comment by 1zun4** [comment]
> Should be good now. Sorry for the bad implementation, this is my first time working with fabric and kotlin

No worries. I will take care of cleaning it up a little.

**Pull request merged**